Backend/app/database/DatabaseConnectionManager.py

```diff
@@ -41,15 +45,38 @@ def get_collection_connection(cls, collection_name: DatabaseCollection) -> Colle
     def init_database_connection(
         cls, environment: AppEnvironmentMode, connection_uri: str
     ) -> None:
-        """Initializes the database connection and loads its unique instance\
-            based on current environment value
+        """
+        Initializes the database connection.
 
         Args:
-            environment (AppEnvironmentMode): the current environment value
-            connection_uri (str): the database connection uri
+            environment (AppEnvironmentMode): The current environment mode.
+            connection_uri (str): The URI for connecting to the database.
+
+        Returns:
+            None
         """
+        from pymongo import MongoClient
+
         database_connection_class = cls.database_connection_mapping.get(
             environment, DatabaseProductionConnection
         )
         database_connection_class.init_connection(connection_uri)
         cls.connection = database_connection_class
+        cls.client = MongoClient(connection_uri)
+
+    @classmethod
+    def ping_database(cls) -> bool:
```

Comment:
We should move this to the concrete implementations of database connection, such as Prod and Testing, included in the same folder